src/BundlerMinifier/MSBuild/BuildBundlerMinifier.targets

```diff
@@ -11,7 +11,7 @@
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerBuildTask"/>
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerCleanTask"/>
 
-  <Target Name="BundleMinify" Condition="'$(RunBundleMinify)' != 'False'">
+  <Target Name="BundleMinify" DependsOnTargets="CompileTypeScript" BeforeTargets="AssignTargetPaths" Condition="'$(RunBundleMinify)' != 'False'">
```

Comment:
Only AfterTargets didn't work. I could try BeforeTargets & AfterTargets. Could test it next week